packages/radio-group/src/LionRadioGroup.js

```diff
@@ -24,102 +25,26 @@ import { LionFieldset } from '@lion/fieldset';
  * @extends {LionFieldset}
  */
 
-export class LionRadioGroup extends LionFieldset {
-  get checkedValue() {
-    const el = this._getCheckedRadioElement();
-    return el ? el.modelValue.value : '';
-  }
-
-  set checkedValue(value) {
-    this._setCheckedRadioElement(value, (el, val) => el.modelValue.value === val);
-  }
-
-  get serializedValue() {
-    const el = this._getCheckedRadioElement();
-    return el ? el.serializedValue : '';
-  }
-
-  set serializedValue(value) {
-    this._setCheckedRadioElement(value, (el, val) => el.serializedValue === val);
-  }
-
+export class LionRadioGroup extends ChoiceGroupMixin(LionFieldset) {
   get formattedValue() {
-    const el = this._getCheckedRadioElement();
+    const el = this._getCheckedElements();
     return el ? el.formattedValue : '';
   }
 
   set formattedValue(value) {
-    this._setCheckedRadioElement(value, (el, val) => el.formattedValue === val);
+    this._setCheckedElements(value, (el, val) => el.formattedValue === val);
   }
 
   connectedCallback() {
     super.connectedCallback();
-    this.addEventListener('model-value-changed', this._checkRadioElements);
     this._setRole('radiogroup');
   }
 
-  disconnectedCallback() {
-    super.disconnectedCallback();
-    this.removeEventListener('model-value-changed', this._checkRadioElements);
-  }
-
-  _checkRadioElements(ev) {
-    const { target } = ev;
-    if (target.type !== 'radio' || target.checked === false) return;
-
-    const groupName = target.name;
-    this.formElementsArray
-      .filter(i => i.name === groupName)
-      .forEach(radio => {
-        if (radio !== target) {
-          radio.checked = false; // eslint-disable-line no-param-reassign
-        }
-      });
-    this.__triggerCheckedValueChanged();
-  }
-
-  _getCheckedRadioElement() {
-    const filtered = this.formElementsArray.filter(el => el.checked === true);
-    return filtered.length > 0 ? filtered[0] : undefined;
-  }
-
-  async _setCheckedRadioElement(value, check) {
-    if (!this.__readyForRegistration) {
-      await this.registrationReady;
-    }
-
-    for (let i = 0; i < this.formElementsArray.length; i += 1) {
-      if (check(this.formElementsArray[i], value)) {
-        this.formElementsArray[i].checked = true;
-        return;
-      }
-    }
-  }
-
+  /**
+   * @override from LionFieldset
+   */
   _onFocusOut() {
     this.touched = true;
     this.focused = false;
   }
```

Comment:
Do you see any point on having this method here? It seems like a duplicate of whats going on in `LionFieldset`.